### ISSUE TITLE
Review CAN_SEE_THROUGH_STRUCTURE on civilian objects

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#1446]
Review CAN_SEE_THROUGH_STRUCTURE on civilian objects. Check if it should be removed or added on some.

For example on the Mogadishu Well it could be added, because the object has low height.

https://user-images.githubusercontent.com/11547761/200130870-234102c0-a0c0-4ffc-8d4a-468c5c3365e1.mp4

**Comment by commy2** [comment]
Apparently and confusingly, CAN_SEE_THROUGH_STRUCTURE enables units to shoot at targets behind buildings.

![https://user-images.githubusercontent.com/11547761/200128992-92080cc0-fe91-432f-88a6-2cefa3ccd77a.png](https://user-images.githubusercontent.com/11547761/200128992-92080cc0-fe91-432f-88a6-2cefa3ccd77a.png)

It's clear from https://github.com/TheSuperHackers/GeneralsGamePatch/pull/1442 that EALA intended to add this flag to some buildings. However, due to an error, it was never added to some of them. 

I think the behaviour should match what a normal adult that has spend less than 1000 hours in this game would be expected visually.

MogadishuWell and GreekRuinXX should be changed such that units can shoot at targets behind them. I am unsure about GLA_StatueSm though. ~SpecialEffectsTrainCrashObject is not depicted in the screenshot as far as I can tell.~

**Comment by Stubbjax** [comment]
> I think the behaviour should match what a normal adult that has spend less than 1000 hours in this game would be expected visually.

I agree with this as well. The idea was for this to be introduced as a separate change, due to the impact on gameplay.

> SpecialEffectsTrainCrashObject is not depicted in the screenshot as far as I can tell.

`SpecialEffectsTrainCrashObject` is an entirely separate case. The two entries were:
```
KindOf = SELECTABLE IMMOBILE BRIDGE
KindOf = SELECTABLE IMMOBILE STRUCTURE BRIDGE LANDMARK_BRIDGE
```
The latter of which completely supersedes the original.

**Comment by commy2** [comment]
Okay, then `SpecialEffectsTrainCrashObject` can be left as is now. The second line only adds two flags that make sense to me, without clobbering anything.
